crates/matrix-sdk/tests/integration/room/joined.rs

```diff
@@ -426,7 +428,7 @@ async fn room_attachment_send_info() {
     }));
 
     let response = room
-        .send_attachment("image", &mime::IMAGE_JPEG, b"Hello world".to_vec(), config)
+        .send_attachment("image", &mime::IMAGE_JPEG, b"Hello world".to_vec(), config, None, None)
```

Comment:
Could you change this test (or introduce a new one that's similar) so that:

- it makes use of a caption here
- the mocked server checks for the right JSON bits that indicate the caption, in the request's JSON? (by modifying the `body_partial_json` line)